config/config.go

```diff
@@ -34,9 +32,8 @@ var (
 )
 
 func initLocalStateDir() {
-	wd, _ := os.Getwd()
 
-	flag.StringVar(&LocalStateDir, "localstatedir", path.Join(wd, "glusterd"), "Directory to store local state information.")
+	flag.StringVar(&LocalStateDir, "localstatedir", "/var/lib/glusterd", "Directory to store local state information.")
```

Comment:
This is something I'm against. The default shouldn't be a directory that would possibly used in a production environment. Doing so can cause frustrations during development.

A default based on the specific system can be set when GD2 is packaged.